### ISSUE TITLE
Icon Request:  AI Sparkle

### ISSUE THREAD

**Issue by lwwells** [google/material-design-icons#1697]
We need an icon for our Flutter app to represent Generative AI.  Most notable companies out there are using a 'sparkle' icon to represent AI.   Can you please add an icon along the lines of these?  It doesn't have to match these exactly, but you get the idea.  

The first is how Google is using the sparkle as part of its cloud AI offerings, and the next two are random pics I found.

![image](https://github.com/google/material-design-icons/assets/13890347/1d53d1ba-4506-40e6-973c-786777113bf9)

![image](https://github.com/google/material-design-icons/assets/13890347/a52ce88f-dbdb-45b4-b6fb-3418afd7b3ad)

![image](https://github.com/google/material-design-icons/assets/13890347/f1548e6c-ee4a-415a-be96-70db72f8552d)

**Comment by tphinney** [comment]
Ah, yes. Google has been keeping its own AI “spark” icon proprietary/internal for branding reasons. But of course there is external demand for such. 

For map pin icons, Google has a different internal version from the external, for this reason. They could do the same for the AI spark.

**Comment by lwwells** [comment]
> Ah, yes. Google has been keeping its own AI “spark” icon proprietary/internal for branding reasons. But of course there is external demand for such.
> 
> For map pin icons, Google has a different internal version from the external, for this reason. They could do the same for the AI spark.

Agreed...if we could have an icon that is somewhat close, that would work...this style of AI indicator is quickly becoming ubiquitous.

**Comment by avipingale** [comment]
Ah, yes. Google has been keeping its own AI “spark” icon proprietary/internal for branding reasons. But of course there is external demand for such.

For map pin icons, Google has a different internal version from the external, for this reason. They could do the same for the AI spark

**Comment by yacinehmito** [comment]
It seems to still be available as `auto_awesome`, but on the website it only shows up the search for Material Icons, although it indeed work with the Material Symbols fonts.

**Comment by lwwells** [comment]
> It seems to still be available as `auto_awesome`, but on the website it only shows up in the search for Material Icons, although it indeed work with the Material Symbols fonts.

Ah, there it is!  Thanks so much for bringing this to my attention.  It is available as auto_awesome for me.  It might be nice to have this reflected on the Material Symbols list.

**Comment by anshukaira** [comment]
Hi, is this icon also available in figma plugins? I am using "Material Symbols" but could not find this anywhere.

**Comment by crs-tna** [comment]
+++

**Comment by jenul-ferdinand** [comment]
AutoAwesome

**Comment by gRodrigues03** [comment]
+2

**Comment by oliverkierepka** [comment]
YES please so i do not need to implement it manually

**Comment by oliver728** [comment]
Please add

**Comment by navimixu** [comment]
yeah that would be super helpful to have in the current day and age, I feel that the spark has became almost a universal symbol for ai, so maybe something basic like a single spark like "assistant" but without the speech bubble.
